### ISSUE TITLE
Multiple slaves working on single transcode?

### ISSUE THREAD

**Issue by iamangus** [wnielson/Plex-Remote-Transcoder#75]
I am in the process of building a docker swarm with storage provided by glusterfs all setup across 4 or 5 Odroid HC2s. My hopes were to fork deevus' docker implementation using linuxserver/docker-plex-armhf so it will run on arm. 

Because I would like to transcode 4K content, is it at all possible to have multiple slaves working on a single transcode?

**Comment by liviynz** [comment]
Hi,
Unfortunately you can’t split a single transcode across multiple slaves without at best a massive rewrite of everything. It wasn’t coded that way because Plex itself deals itself in single streams so the code is aligned with that. Even if it was redone then there’s a likely issue of the playback halting if one of the slave transcodes don’t keep up. I believe what I’ve said here is all accurate but my focus is elsewhere with some other non PRT critical issues and I just happened to be taking a 10 min break when this came through so I thought I’d quickly reply and the subject of splitting the transcoding of a single job across multiple systems simultaneously has come up before. 

TLDR version, plex is single stream per transcode orientated so therefor so is PRT. Is it possible? Yes potentially but large amounts of work & lots of potential problems so it’s not planned currently.
